mods/ts/weapons/healweapons.yaml

```diff
@@ -1,25 +1,25 @@
-Heal:
+^HealWeapon:
 	ReloadDelay: 80
 	Range: 2c849
 	Report: healer1.aud
-	ValidTargets: Infantry
-	Projectile: Bullet
-		Speed: 1c682
+	Projectile: InstantHit
 	Warhead@1Dam: SpreadDamage
```

Comment:
The way I would do it is to have the warhead check the `Target` type, and if it's an actor then only apply damage to that actor.  If it's a WPos then search for all actors with an overlapping hit shape, and pick the one that is closest (or all?)